### ISSUE TITLE
Combine ip and port in the request

### ISSUE THREAD

**Issue by shdf2** [samuel-allan/FastProxyChecker#5]
Hello,
why not making the request that way, this would be easier to handle : 
checker.php?ip=1.1.175.47:8080&timeout=10
instead of 
checker.php?ip=1.1.175.47&port=8080&timeout=10

Or maybe you can just add the possibility with a new variable $_GET["ipport"], that way you can support both methods : 
checker.php?ipport=1.1.175.47:8080&timeout=10
checker.php?ip=1.1.175.47&port=8080&timeout=10

**Comment by samuel-allan** [comment]
Good idea. Do you want to incorporate this? If not I'll do it about next
week.
